cmd/testcase_validate.go

```diff
@@ -110,11 +110,16 @@ func runTestCaseValidateArg(cmd *cobra.Command, client *api.Client, fileOrStdin
 		return !success, nil
 	}
 
-	errorMeta, err := api.UnmarshalErrorMeta(strings.NewReader(message))
+	errorMeta, err := api.ErrorDecoder{SourceMapper: mapper}.UnmarshalErrorMeta(strings.NewReader(message))
 	if err != nil {
 		return true, err
 	}
 
+	if len(errorMeta.Errors) > 0 && errorMeta.Errors[0].EvaluationErrorMeta != nil {
```

Comment:
@tisba This can probably be deleted, right? Feels like this was just for debugging?

Reply:
yes, just to poke around 👍

Reply:
🔥